assets/i18n/strings/de_AT.xlb

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is Austria intentionally included in the Japan PR?

Reply:
Good catch, I'll create a new PR for that